ccp/lib/clients/jenkins/workflow_client.py

```diff
@@ -122,8 +126,11 @@ def get_logs_of_execution_node(
         Lets the logs of a jenkins execution node, if possible
         :param job_ordered_list: The ordered list of jobs, with parents,
         followed by children
+        :type job_ordered_list: list
         :param build_number: The number of the build to describe.
+        :type build_number: str
         :param node_number: The number of the node to describe
+        :type node_number: str
```

Comment:
I reviewed the PR commit by commit, since you have fixed it next commit, ignore the earlier comments on this.